solenspipe/solenspipe.py

```diff
@@ -121,14 +121,14 @@ def get_tempura_norms(est1,est2,ucls,tcls,lmin,lmax,mlmax):
         Nl_phi_c = Als[e1][1]*Als[e2][1]*R_e1_e2[1]
         Nl_g = Nl_phi_g * (ls*(ls+1.)/2.)**2.
         Nl_c = Nl_phi_c * (ls*(ls+1.)/2.)**2.
-        if bh:
+        if bh and est2=='SRC':
             Nl_g_bh = bias_hardened_n0(Nl_phi_g,Als['src'],R_src_tt) * (ls*(ls+1.)/2.)**2.
         else:
             Nl_g_bh = None
     return bh,ls,Als,R_src_tt,Nl_g,Nl_c,Nl_g_bh
 
 
-def get_qfunc(px,ucls,mlmax,est1,Al1=None,est2=None,Al2=None,R12=None):
+def get_qfunc(px,ucls,mlmax,est1,Al1=None,est2=None,Al2=None,R12=None,profile=None):
```

Comment:
Looks good - just need to add the profile argument to the Parameters list in the docstring